### PR TITLE
Use Deno global instead of module

### PR DESCRIPTION
**Pull request by sh7dm**
/cc @ry

### CODE REVIEW

**Comment by keroxp** [comment]
I think this PR should be opened to `v0.3` branch with other major update features. @ry How dou you think?

**Comment by ry** [comment]
I think we should be able to land it now - with a small fix. I don't want to try to branch now, but rather break code. 

Maintain branches is a lot of overhead that I don't want to take on now. Users have been warned: https://deno.land/manual.html#disclaimer
We'll invest more in stable APIs as the project matures - but for now we live at edge.

File: http/file_server.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   setContentLength,
   ServerResponse
 } from "./server.ts";
-import { cwd, DenoError, ErrorKind, args, stat, readDir, open } from "deno";
+const { cwd, DenoError, ErrorKind, args, stat, readDir, open } = Deno;
 import { extname } from "../fs/path.ts";
```
Comment (ry) [inline http/file_server.ts]: For the time being add this:
```ts
// TODO https://github.com/denoland/deno/issues/1810
import { DenoError } from "deno";
```

**Comment by ry** [comment]
@sh7dm please rebase

```
git fetch origin
git rebase -i master
```

**Comment by ry** [comment]
@sh7dm can you get this to go green?

**Comment by ry** [comment]
fixed in https://github.com/denoland/deno_std/pull/218

**Pull request closed without merging**